### ISSUE TITLE
cannot scroll braille after using command to read character

### ISSUE THREAD

**Issue by mehgcap** [nvaccess/nvda#8682]
### Steps to reproduce:

1. Connect a braille display. I'm using an Orbit Reader 20 connected via USB.
2. Open an application that allows character review. I've tested a webpage on Chrome and a file in Notepad Plus Plus. I suspect anywhere will work, though.
3. Make sure you can scroll. Your text should be more than the length of your display, and your scroll buttons should move display width by display width as normal.
4. Now press NVDA-period (I'm using the laptop keyboard layout).
5. Now try to scroll using the panning keys on your display, the ones you verified worked in a previous step.

### Actual behavior:
Braille does not pan. The display remains on the current line until you move the cursor with the keyboard.

### Expected behavior:
Braille should pan as normal.

#### NVDA Installed/portable/running from source:
Installed.

#### NVDA version:
2018.2.1

#### Windows version:
Windows 10, v1803

#### Name and version of other software in use when reproducing the issue:
Chrome, Notepad Plus Plus, suspect software doesn't matter

#### Other information about your system:
Nothing pertinent.

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA?
No.

**Comment by leonardder** [comment]
cc @dkager

**Comment by dkager** [comment]
I can only reproduce this:
1. outside browse mode; and
2. with braille tethered automatically.

**Comment by mehgcap** [comment]
I can get this to happen regardless of the browse mode status. It happens on webpages when arrowing through them, and it happens in edit fields when editing. You're right, though, that tethering to focus does make the issue disappear. Only automatic seems to be affected.

**Comment by mehgcap** [comment]
Update: this issue is still present in NVDA 2018.3.1, with the same display setup as in the original report.

**Comment by leonardder** [comment]
The question is what's the bug here:

1. When you press NVDA+period, braille automatically tethers to review, since the script deals with reporting the character where the review cursor is situated, this may be expected from auto tethering.
2. Now if your review cursor is in object review, scrolling should still work within notepad++, but it should move the review cursor, not the physical caret. When in browse mode and you are in object review, it is expected that you can't scroll beyond the edges of the object.

Basically, the scrolling behaviour after pressing NVDA+period should be similar to the review cursor next/previous line commands, until you explicitly move your physical caret again.

**Comment by Adriani90** [comment]
@mehgcap how is the behavior in NVDA 2019.3 Beta 1?
